app/templates/core/components/page-errors.jade

```diff
@@ -59,8 +59,8 @@
           a(href="/") {{ $t("nav.home") }}
         li
           a(v-bind:href="forumLink") {{ $t("nav.forum") }}
-        li(v-if="isAnonymous")
-          a.contact-modal(tabindex=-1) {{ $t("nav.contact") }}
+        //- li(v-if="isAnonymous")
```

Comment:
Is it worth deleting this or do we plan to enable this in the future?

Reply:
Not sure how we would approach it in the future. I commented it since its a small piece of code and thought would be helpful for context in the future.